js/ui/overlay/ui.overlay.js

```diff
@@ -488,6 +489,11 @@ const Overlay = Widget.inherit({
     _normalizePosition: function() {
         const position = this.option('position');
         this._position = typeof position === 'function' ? position() : position;
+        const container = this.option('container');
```

Comment:
are u sure that we should use `this.option('container')` here, not `this._getContainer()`? We don't take into account case when `position.of = #someContainer`. It's ok?

Reply:
It is ok, position.of does not control dom element where the popup is rendered